toolbox/bin/variables.sh

```diff
@@ -58,7 +58,11 @@ fi
 # populate current layer info
 LAYERS_DIR="$WORKING_DIR/layers"
 LAYERS=''
-if [ -d "$LAYERS_DIR" ]; then
+if [ ! -d "$LAYERS_DIR" ]; then
+  # CircleCI fails the build if none of the files mentioned in persist_to_workspace exist.
+  # changed_layers is the only one that makes to create in this situation
+  echo > "$WORKSPACE_DIR/changed_layers"
```

Comment:
If you just touch changed_layers this wouldn't need to be conditional is that right?